src/plugins/jartic/archive_downloader.py

```
@@ -0,0 +1,241 @@
+            file_stats = archive_path.stat()
+
+            with zipfile.ZipFile(archive_path, 'r') as zf:
+                file_list = zf.namelist()
+                total_uncompressed = sum(zf.getinfo(f).file_size for f in file_list)
+
+            return {
+                'path': str(archive_path),
+                'size_mb': file_stats.st_size / 1024 / 1024,
+                'uncompressed_size_mb': total_uncompressed / 1024 / 1024,
+                'file_count': len(file_list),
+                'modified': datetime.fromtimestamp(file_stats.st_mtime)
```

Ruff (0.12.2)

`datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)